app/models/miq_schedule.rb

```diff
@@ -39,6 +42,11 @@ class MiqSchedule < ApplicationRecord
   default_value_for :enabled, true
   default_value_for(:zone_id) { MiqServer.my_server.zone_id }
 
+  def resource
+    # HACK: this should be a real relation, but for now it's using a reserve_attribute for backport reasons
+    Object.const_get(towhat).find_by(:id => resource_id)
```

Comment:
Are we deprecating this here?  The commit says we are but maybe it's coming in a future PR?